src/rviz/default_plugin/image_display.h

```diff
@@ -98,6 +101,8 @@ public Q_SLOTS:
   FloatProperty* max_property_;
   IntProperty* median_buffer_size_property_;
   bool got_float_image_;
+
+  MouseClick* mouse_click;
```

Comment:
Please follow the naming convention for class members:
```suggestion
  MouseClick* mouse_click_;
```

Reply:
This variable rename requires corresponding changes in other places as well!

Reply:
Done. I have corrected and it compiles.